### PR TITLE
Refactoring client: separate files, use repository pattern

### PR DESCRIPTION
**Pull request by tkmct**
Instead of accessing directly to database from light client, use repository pattern for better code separation.
It will make it easier to implement Usecase classes and Dispute classes.
Separate usecase classes

### CODE REVIEW

File: packages/plasma-light-client/src/types/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './Nuberish'
```
Comment (syuhei176) [inline packages/plasma-light-client/src/types/index.ts:1]: Numberish?

Comment (tkmct) [reply packages/plasma-light-client/src/types/index.ts]: Thanks! Fixed.

**Comment by syuhei176** [comment]
code in e2e-tests is using `clients["stateManager"]` and `clients["createExit"]` and integration tests don't pass.
https://github.com/cryptoeconomicslab/gazelle/blob/master/integration-test/e2e-tests/__tests__/index.test.ts#L494

**Comment by tkmct** [comment]
Sure. I'll fix this ASAP.

**Pull request merged**